crouton.php

```diff
@@ -420,30 +419,30 @@ public function renderMap($atts)
                     ? boolval($atts['map_search_coordinates_search'])
                     : $this->shortCodeOptions['map_search_coordinates_search']
             ];
-            return $this->getInitializeCroutonBlock($this->getCroutonJsConfig($atts));
+            return $this->getInitializeCroutonBlock(...$this->getCroutonJsConfig($atts));
         }
 
-        public function initCrouton($atts)
+        private function initCrouton($atts)
         {
-            return $this->getInitializeCroutonBlock($this->getCroutonJsConfig($atts));
+            return $this->getInitializeCroutonBlock(...$this->getCroutonJsConfig($atts));
         }
 
         public function meetingCount($atts)
         {
             $random_id = rand(10000, 99999);
-            return $this->getInitializeCroutonBlock($this->getCroutonJsConfig($atts)) . "<script type='text/javascript'>jQuery(document).ready(function() { crouton.meetingCount(function(res) { document.getElementById('meeting-count-$random_id').innerHTML = res; }) })</script><span id='meeting-count-$random_id'></span>";
+            return $this->getInitializeCroutonBlock(...$this->getCroutonJsConfig($atts)) . "<script type='text/javascript'>jQuery(document).ready(function() { crouton.meetingCount(function(res) { document.getElementById('meeting-count-$random_id').innerHTML = res; }) })</script><span id='meeting-count-$random_id'></span>";
         }
 
         public function groupCount($atts)
         {
             $random_id = rand(10000, 99999);
-            return $this->getInitializeCroutonBlock($this->getCroutonJsConfig($atts)) . "<script type='text/javascript'>jQuery(document).ready(function() { crouton.groupCount(function(res) { document.getElementById('group-count-$random_id').innerHTML = res; }) })</script><span id='group-count-$random_id'></span>";
+            return $this->getInitializeCroutonBlock(...$this->getCroutonJsConfig($atts)) . "<script type='text/javascript'>jQuery(document).ready(function() { crouton.groupCount(function(res) { document.getElementById('group-count-$random_id').innerHTML = res; }) })</script><span id='group-count-$random_id'></span>";
         }
 
         public function serviceBodyNames($atts)
         {
             $random_id = rand(10000, 99999);
-            return $this->getInitializeCroutonBlock($this->getCroutonJsConfig($atts)) . "<script type='text/javascript'>jQuery(document).ready(function() { crouton.serviceBodyNames(function(res) { document.getElementById('service-body-names-$random_id').innerHTML = res; }) })</script><span id='service-body-names-$random_id'></span>";
+            return $this->getInitializeCroutonBlock(...$this->getCroutonJsConfig($atts)) . "<script type='text/javascript'>jQuery(document).ready(function() { crouton.serviceBodyNames(function(res) { document.getElementById('service-body-names-$random_id').innerHTML = res; }) })</script><span id='service-body-names-$random_id'></span>";
```

Comment:
this is fine as we did it before but maybe in the future we look into using wp `wp_add_inline_script` function

Reply:
You are right.  But could I mark it as a TODO and do it in 3.17.2, as I think this is a big enough change to require another beta, and it won't be so long until 3.17.2, anyway.